lib/src/react16_suggestors/constants.dart

```diff
@@ -19,9 +19,16 @@ const manualValidationCommentSubstring = 'Check this box upon manual';
 const willBeRemovedCommentSuffix =
     ' This will be removed once the transition to React 16 is complete.';
 
-const styleMapExample = '''
-    // CSS number strings are no longer auto-converted to px. Ensure values are of type `num`, or have units.
-    // Incorrect value for 'width': '40'. Correct values: 40, '40px', '4em'.''';
+const styleMapExplanation = '// CSS number strings are no longer '
+    'auto-converted to px. Ensure values are of type `num`, or have units.';
+
+const styleMapExample = '// Incorrect value for \'width\': \'40\'. Correct'
+    'values: 40, \'40px\', \'4em\'.';
```

Comment:
This is pretty hard to read; I'm a fan of double quotes when there are a lot of single quotes. Also, you don't always have to split up strings; it's okay if they occasionally go past the line length:

```dart
const styleMapExample = 
    "// Incorrect value for 'width': '40'. Correct values: 40, '40px', '4em'.";
```


Another option is to use the multiline string behavior where leading newlines are automatically trimmed:
```dart
const styleMapExample = '''
// Incorrect value for 'width': '40'. Correct values: 40, '40px', '4em'.'''
```

Reply:
I had a default Webstorm setting on that autowrapped at 80. Usually it wasn't a big deal and didn't get in my way, but it's caused a handful of fun formatting moments lately.. so I turned that off haha.

But nice, good to know they can run passed the line length every now and then!